frontend/src/config/shared/layers.json

```diff
@@ -976,49 +976,5 @@
     "opacity": 0,
     "legend_text": "",
     "legend": []
-  },
-  "adamts_buffers": {
```

Comment:
Do you remember the rationale for deleting this from the shared files?

Reply:
Ok, so we keep it like this: everything in mozambique layers.json

Note that adamts_buffers is mentioned in other country configs:

![Uploading Screenshot 2025-07-14 at 3.02.09 PM.png…]()

Reply:
@ericboucher, I just checked on this. We can keep config in shared, but we do need to make each country unique for the adamts_buffers layer. For example, Myanmar's layers.json file can delete `adamts_nodes`, and `adamts_tracks` and just keep these lines for `adamts_buffers`:

```
  "adamts_buffers": {
    "server_layer_name": "mmr_gdacs_buffers",
    "content_path": "data/myanmar/contents.md#tropical-storm-wind-buffers"
  },
```